intake_nested_yaml_catalog/tests/zestimate.csv

```diff
@@ -0,0 +1,2 @@
+zpid,zestimate
```

Comment:
as a suggestion I would choose non-internal zillow constructs. These seem benign but I would suggest making it a practice to scrub zillow-specific terminology and references.